addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/PiiScanRuleUnitTest.java

```diff
@@ -49,6 +51,26 @@
 /** Unit test for {@link PiiScanRule}. */
 class PiiScanRuleUnitTest extends PassiveScannerTest<PiiScanRule> {
 
+    private static final String CONTENT_INSIDE_SCRIPT_TAG =
+            "<html><body><script>var data = '{\"identifier\": \"GRID_LAYOUT_4743251660071187\"}';</script></body></html>";
+    private static final String CONTENT_INSIDE_IMG_ID_ATTR =
+            "<html><body><img id=\"GRID_LAYOUT_4743251660071187\" src=\"image.png\"></img></body></html>";
+    private static final String CONTENT_INSIDE_H1_CLASS_ATTR =
+            "<html><body><H1 class=\"GRID_LAYOUT_4743251660071187\">Heading One</H1></body></html>";
+    private static final String CONTENT_INSIDE_HTML_COMMENT =
+            "<html><body><!-- Example comment GRID_LAYOUT_4743251660071187 --><H1>Heading One</H1></body></html>";
+    private static final String CONTENT_SCRIPT_BLOCK_PLACEHOLDER =
```

Comment:
Do we really need all these individually? A single HTML with all of the elements would be enough IMO (especially since the behaviour is of a third party).